pkg/apis/agones/v1/gameserver.go

```diff
@@ -198,6 +202,15 @@ type PlayersSpec struct {
 	InitialCapacity int64 `json:"initialCapacity,omitempty"`
 }
 
+// CountersSpec tracks if counter specified (for giving error message if feature gate not set)
+type CountersSpec struct {
```

Comment:
nit: should this be `CounterSpec`? since it represents one Counter. Similar for `ListsSpec`